src/Core/Components/List/FluentPersona.razor

```diff
@@ -2,23 +2,26 @@
 @inherits FluentComponentBase
 
 <div id="@Id" class="@ClassValue" style="@StyleValue" @attributes="@AdditionalAttributes">
-    <div class="initials" @onclick="@OnClick" style="@(OnClick.HasDelegate ? $"cursor: pointer;" : null)">
-        <FluentPresenceBadge Status="@Status" Size="@StatusSize" StatusTitle="@StatusTitle">
-            <div style="display: table-cell; vertical-align: middle; @GetImageMinSizeStyle()">
-                @if (string.IsNullOrEmpty(Image))
-                {
-                    @(Initials ?? GetDefaultInitials())
-                }
-                else
-                {
-                    <img src="@Image" alt="@Name" title="@Name" style="@GetImageMaxSizeStyle()" />
-                }
-            </div>
-        </FluentPresenceBadge>
-    </div>
+    @if (!ImagePosition.HasValue || ImagePosition == PersonaImagePosition.Left)
```

Comment:
To avoid duplicating code in this file (depending on the value of `ImagePosition`), you could use CSS like this, based on a new `position` attribute to be defined on the `div` "main".

```xml
<div id="@Id" class="@ClassValue" position="@ImagePosition.ToAttributeValue()" ...
```

```css
.fluent-persona[position='before'] .name {
    order: 1;
    margin-inline-end: 16px;
}

.fluent-persona[position='before'] .close {
    order: 2;
}

.fluent-persona[position='before'] .initials {
    order: 3;
}
``` 

What do you think? Possible (to be verified if no side effects are detected).

Reply:
Done. It is cleaner. We might have to revisit this if the goal is feature-parity with fluentui react.